src/main/java/core/basesyntax/JavaDateTimeApi.java

```diff
@@ -74,7 +101,11 @@ public LocalDate addWeeks(LocalDate localDate, Integer numberOfWeeks) {
      * - "someDate is today" - если someDate - сегодня
      */
     public String beforeOrAfter(LocalDate someDate) {
-        return someDate + "is today";
+        return LocalDate.now().compareTo(someDate) == 0
```

Comment:
let's store `LocalDate.now()` in a local variable
and use `isAfter()` and `isBefore()` methods